R/dl_build_src_gran.R

```diff
@@ -4,104 +4,120 @@
 #' 
 #' @param GRAN.dir local directory for GRAN built packages
 #' @param lib local directory to install dependent packages
+#' @param delete logical argument to over-ride deleting packages that 
+#' are already built. This is useful for major R upgrades. Default
```

Comment:
So use `delete = FALSE` for major R upgrades?

Reply:
exactly